components/automate-chef-io/Makefile

```diff
@@ -3,12 +3,9 @@
 SHELL=bash
 SWAGGER_RESULT_FILE=static/api-docs/all-apis.swagger.json
 SWAGGER_DIR=data/docs/api_chef_automate
-#SWAGGER_FILES=$(shell find $(SWAGGER_DIR) -name '*.swagger.json')
-SWAGGER_FILES=data/docs/api_chef_automate/compliance/reporting/reporting.swagger.json
+SWAGGER_FILES=$(shell find $(SWAGGER_DIR) -name '*.swagger.json')
 STATIC_SWAGGER_FILES=$(shell find data/docs/api-static/ -type f | sort -n)
 
-EXPECTED_CONFLICTS=46
```

Comment:
🎉

Reply:
I think this was mostly "solved" by `jq` eating the return value of the `swagger` call, unfortunately:

```
Petes-MacBook-Pro:automate-chef-io pete$ ruby -rjson -e "puts JSON.dump({foo: 'bar'}) ; exit 1" ; echo $?
{"foo":"bar"}
1
Petes-MacBook-Pro:automate-chef-io pete$ ruby -rjson -e "puts JSON.dump({foo: 'bar'}) ; exit 1" | jq . ; echo $?
{
  "foo": "bar"
}
0
```